clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -617,11 +632,6 @@
   MatchMode: Partial
   In: include_directories(${value})
   Out: include_directories(${value})
```

Comment:
Why sub rule is removed here？

Reply:
The keyword CUDA_INCLUDE_DIRS can be used in other CMake macros besides include_directories like - target_include_directories, set etc.

Hence, the subrule has been removed added as a new separate rule in [L#599](https://github.com/oneapi-src/SYCLomatic/pull/1816/files#diff-5cdee37b283c4333ea78cb8b32fb827505a7cb7c91ec89bb2a1e0df87862bf69R599)

Reply:
Since this rule is duplicated with [L#599](https://github.com/oneapi-src/SYCLomatic/pull/1816/files#diff-5cdee37b283c4333ea78cb8b32fb827505a7cb7c91ec89bb2a1e0df87862bf69R599), this rule could also be removed.

Reply:
Okay, I'll remove it